src/monopoly/view/monopoly_view.js

```diff
@@ -911,18 +989,44 @@ MonopolyView.prototype.updateMessage = function() {
 
     switch (this.game.state) {
 
-      case BUY_ANSWER:
-
-        this.button1Text.text = "Yes";
-        this.button2Text.text = "No";
-        this.button2.alpha = 1;
-        break;
-
-      default:
-        this.button1Text.text = "Continue";
-        this.button2Text.text = "";
-        this.button2.alpha = 0;
-        break;
+        case BUY_ANSWER:
+
+            this.button1Text.text = "Yes";
+            this.button2Text.text = "No";
+            this.button2.alpha = 1;
+            this.button3.alpha = 0;
+            break;
+
+        case TRADE_ANSWER:
+            this.button1Text.text = "Yes";
+            this.button2Text.text = "No";
+            this.button2.alpha = 1;
+            this.button3.alpha = 0;
+            break;
+
+        case POST_TURN_ANSWER:
+
+            this.button1Text.text = "Continue";
+            this.button2Text.text = "Trade";
+            this.button2.alpha = 1;
+            this.button3.alpha = 0;
+            break;
+
+        case PROPOSE_TRADE:
+
+            this.updateTradeInfo();
+            this.button1Text.text = "Continue";
+            this.button2Text.text = "Clear";
+            this.button2.alpha = 1;
+            this.button3.alpha = 1;
+            break;
+
+        default:
+            this.button1Text.text = "Continue";
+            this.button2Text.text = "";
+            this.button2.alpha = 0;
+            this.button3.alpha = 0;
+            break;
```

Comment:
should abstract these to a function, ie: 

    MonopolyView.prototype.updateButtons = function(buttonsText, buttonsAlpha) {
          this.button1Text.text = buttonsText[0];
          ...
    }